spec.html

```diff
@@ -4393,6 +4393,42 @@ <h1>The Lexical Environment and Environment Record Specification Types</h1>
       <p>The Lexical Environment and Environment Record types are used to explain the behaviour of name resolution in nested functions and blocks. These types and the operations upon them are defined in <emu-xref href="#sec-lexical-environments"></emu-xref>.</p>
     </emu-clause>
 
+    <emu-clause id="sec-abstract-closure">
+      <h1>The Abstract Closure Specification Type</h1>
+      <p>The <dfn>abstract closure</dfn> specification type is used to refer to algorithmic steps together with a collection of values. Abstract closure are meta-values and are invoked using function application style such as _closure_(_arg1_, _arg2_). Like abstract operations, invocations perform the algorithmic steps described by the abstract closure. Unlike abstract operations, abstract closures are meta-values and must be created.</p>
+      <p>In algorithmic steps describing creating an abstract closure, values are captured with the verb "capture". Note that only values are capturable, not references. In algorithmic steps describing calling an abstract closure, the captured values are referred to with their captured name.</p>
+      <p>Abstract closures may be created inline as part of other algorithms, shown in the following example.</p>
+
+      <emu-alg>
+        1. Let _addend_ be *41*.
+        1. Let _closure_ be a new abstract closure with parameters (_x_) that captures _addend_ and performs the following steps when called:
+          1. Return _x_ + _addend_.
+        1. Let _val_ be _closure_(*1*).
+        1. Assert: _val_ is *42*.
+      </emu-alg>
+
+      <p>Abstract closures may also be defined like abstract operations. When defined independently, the notational convention is to use two parameter lists. The first parameter list is for creating an abstract closure. The second parameter list is for calling the created abstract closure meta-value. Please see below for an example.</p>
+
+      <emu-clause id="sec-exampleconstant" aoid="ExampleConstant">
+        <h1>ExampleConstant ( _value_ ) ( )</h1>
+        <p>Creating an ExampleConstant abstract closure takes an argument _value_ and performs the following steps:</p>
```

Comment:
This part is a bit weird - we don't have anywhere else where this "Creating an X performs the following steps" construction exists, do we?

Reply:
Also nothing uses this form, as of this PR; perhaps it could be introduced later when needed?

Reply:
Yeah let's remove it.